schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaOperator.java

```diff
@@ -85,6 +86,31 @@ public void createTables(List<TableSchema> tableSchemaList) throws SchemaLoaderE
     }
   }
 
+  public void repairNamespaces(List<TableSchema> tableSchemaList) throws SchemaLoaderException {
+    // Arbitrarily use the configuration of the first table listed in the schema for the reparation
+    // That means if tables of a namespace have different options (resource unit, replication
+    // factor, etc.), the options of the first table will be used for the reparation
+    Map<String, TableSchema> namespaceToTableSchema = new HashMap<>();
+    for (TableSchema tableSchema : tableSchemaList) {
+      namespaceToTableSchema.putIfAbsent(tableSchema.getNamespace(), tableSchema);
```

Comment:
In my understanding, `transactionAdmin`'s `repairNamespace` is a superset of `storageAdmin`'s one. If so, how about overwriting a value if a previous table schema isn't transaction table and a new one is transaction table?

Reply:
Right, I will do that, thanks.

Reply:
https://github.com/scalar-labs/scalardb/pull/1203/commits/d3c00d280e4d8f3da9f07c4ba2e01279702d94f1